.github/workflows/checks.yml

```diff
@@ -32,14 +32,21 @@ jobs:
       - name: npm install
         run: npm install
       - name: run eslint
+        if: github.event.pull_request.head.repo.fork == true
+        run: npm run lint
```

Comment:
Currently, the `lint` npm script also executes `tsc`, i.e. type checking. This is already done in the other job. We should probably ad a script that _only_ executes eslint, maybe call it `lint:ci` or so.

Also I don't think a difference between running it from a fork and running it from the base repo should be introduced. 

Personally, I don't think there is much value in having the lint results being uploaded. The foundry-0.7.9 branch is completely green anyways, so what's the point...? I mean seriously, has anybody ever looked at those results in GitHub? :D

So personally I would completely ditch it in favor of simply running a job like `npm run lint:ci` or similar. In my eyes, the CI is only the gate keeper to prevent bad code from accidentally being merged. You can always look at the concrete issues locally (which you will most likely do anyways).

Reply:
I would agree. Since you all have husky hooked up as well, I actively had to try to upload bad code around it.

Reply:
(you can just remove the lint-staged configuration in the `package.json` when commiting, but psssst, don't tell anybody... :zipper_mouth_face: )